backend/app/Http/Controllers/Api/VisitorController.php

```diff
@@ -29,4 +36,15 @@ public function getAllVisitors()
             ])
         );
     }
+
+    public function getNewVisitors()
+    {
+        $response = $this->getNewVisitorsAction->execute();
+
+        return ApiResponse::success(
+            new GetNewVisitorsResponse([
```

Comment:
Did you implement this response? It should return empty response, but work anyway. Write a test on this endpoint

Reply:
Response was checked and work well. I will write a test later